pkg/controllers/job/plugins/ssh/ssh.go

```diff
@@ -121,7 +115,7 @@ func (sp *sshPlugin) mountRsaKey(pod *v1.Pod, job *batch.Job) {
 				Path: SSHRelativePath + "/" + SSHPublicKey,
 			},
 			{
-				Key:  SSHPublicKey,
+				Key:  SSHAuthorizedKeys,
```

Comment:
And here the authorized_keys file should be from configmap's SSHAuthorizedKeys, though SSHPublicKey and SSHAuthorizedKeys are of same value